setup.py

```diff
@@ -25,6 +25,7 @@
     "ipython>=7.1.1",
     "numba==0.45.0",  # numba 0.45.1 has a conflict with UMAP and numba 0.46.0 with parallelization in loompy
     "hyperopt==0.1.2",
+    "sparse>=0.8.0",
```

Comment:
can't we use scipy.sparse ?

Reply:
SciPy.sparse only allows 2d. We need 3d. This is a pretty nice package. We can move this to criticism_requirements if necessary. It's a pretty easy install either way since these are the only requirements
numpy
scipy>=0.19
numba>=0.45

Reply:
I see, alright